spec/lib/plugin/instance_spec.rb

```diff
@@ -324,49 +324,6 @@ def enabled?
     end
   end
 
-  it "patches the enabled? function for auth_providers if not defined" do
```

Comment:
This test in its entirety is there to test the backwards compatibility patch, so we can remove it.